frontend/src/modules/EditZing/Components/EmailModalContent.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react'
+import { EmailModalContentProps } from '../Types/ComponentProps'
+import {
+  ModalContainer as ContentContainer,
+  RecipientsContainer as InfoContainer,
+} from '../Styles/Email.style'
+import { EmailTemplateButtons } from './EmailTemplateButtons'
+import { EmailPreview } from './EmailPreview'
+import { Box, Button, Typography } from '@mui/material'
+import { TemplateName } from 'EditZing/utils/emailTemplates'
+import SendIcon from '@mui/icons-material/Send'
+
+export const EmailModalContent = ({
+  selectedGroups,
+  courseNames,
+  isEmailing,
+  setIsEmailing,
+}: EmailModalContentProps) => {
+  const [selectedTemplate, setSelectedTemplate] = useState<TemplateName>(
+    '' as TemplateName
+  )
+  const [step, setStep] = useState<number>(0)
+  const titles = ['Select email template', 'Send emails']
+
+  const RecipientsComponent = () => {
+    let groupNames = ''
+    selectedGroups.forEach((group, index) => {
+      if (index !== selectedGroups.length - 1) {
+        groupNames += `Group ${group.groupNumber}, `
+      } else {
+        groupNames += `Group ${group.groupNumber}`
+      }
+    })
+    return (
+      <InfoContainer>
+        <Typography variant="h5" component="h5">
+          To:
+        </Typography>
+        &nbsp;&nbsp;
+        <Typography variant="h5" component="h5" fontWeight={'900'}>
+          {groupNames}
+        </Typography>
+      </InfoContainer>
+    )
+  }
+
+  const TemplateSelectedComponent = () => {
+    return (
+      <InfoContainer style={{ marginTop: '0.25em', marginBottom: '0.5em' }}>
```

Comment:
Don't feel like InfoContainer/RecipientContainer really needs to be its own reusable component if one of the two places where it's reused, it has style overrides anyway (plus it's just a `display: flex`? Use a Box?